build.gradle

```diff
@@ -473,7 +473,8 @@ githubRelease {
     body ""
     draft true
     prerelease false
-    releaseAssets file("${project.buildRpmDir}/${project.readableName}-${project.version}.noarch.rpm"),
+    releaseAssets file("${project.buildDir}/libs/${project.readableName}-${project.version}.jar"),
```

Comment:
Could we use tasks.jar.destinationDirectory... (I do not know the right properties)?